A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js

```diff
@@ -320,6 +336,69 @@ class A320_Neo_CDU_MainDisplay extends FMCMainDisplay {
         }
     }
 
+    forceClearScratchpad() {
+        this.setVarOnResponse();
+        this.inOut = "";
+        this.lastUserInput = "";
+        this.isDisplayingErrorMessage = false;
+    }
+
+    /**
+     * Add Type II Message
+     * @param message {string} Message to be displayed
+     * @param color {string} Color of Message
+     * @param f {function} Function gets executed when error message has been cleared
+     */
+    addTypeIIMessage(message, color = "#ffffff", f = () => {}) {
```

Comment:
Can we change this to `addTypeTwoMessage` ?

Reply:
yep